src/test/parse-fail/doc-in-trait-below-keyword.rs

```diff
@@ -0,0 +1,21 @@
+// Copyright 2012 The Rust Project Developers. See the COPYRIGHT
+// file at the top-level directory of this distribution and at
+// http://rust-lang.org/COPYRIGHT.
+//
+// Licensed under the Apache License, Version 2.0 <LICENSE-APACHE or
+// http://www.apache.org/licenses/LICENSE-2.0> or the MIT license
+// <LICENSE-MIT or http://opensource.org/licenses/MIT>, at your
+// option. This file may not be copied, modified, or distributed
+// except according to those terms.
+
+// compile-flags: -Z parse-only
+
+trait Boom {
+    fn kaboom();
+    /// This comment shouldn't throw an error!
```

Comment:
ah, whoops, it should. That was because I was first testing the whole thing to not throw an error :)